core/dbt/clients/_jinja_blocks.py

```diff
@@ -357,8 +357,8 @@ def find_blocks(self, allowed_blocks=None, collect_raw_data=True):
 
         if self.current:
             dbt.exceptions.raise_compiler_error((
-                'Reached EOF without finding a close block for '
-                '{0.block_type_name} (from {0.end})'
+                'Reached EOF without finding a close tag for '
+                '{0.block_type_name} (searched from position {0.end})'
```

Comment:
Do you happen to know if we get a line number in `self.current` here? If so, I think that would be better than a character position. If we can't easily get a line number (what line number would it even be? The end tag doesn't exist...?), then I think we can just delete the text from `(search from ...` onwards.

Reply:
The line number would be the line where we _started_ looking for the end tag:
```
$ cat models/docs.md
{% docs some_doc %}
some stuff
{% enddocs %}

{% docs my_doc %}
Check out my super cool doc() documentation
$ dbt compile
Running with dbt=0.15.2-a1
Encountered an error:
Compilation Error in models/docs.md
  Reached EOF without finding a close tag for docs (searched from line 5)
```